vivisect/symboliks/analysis.py

```diff
@@ -542,7 +542,7 @@ def codewalker(ppath, edge, path):
             patheffs.extend(neweffs)
             return True
 
-        for pathnode in viv_graph.walkCodePaths(graph, codewalker, maxpath=maxpath):
+        for pathnode in viv_graph.walkCodePaths(graph, codewalker, loopcnt=loopcnt, maxpath=maxpath):
```

Comment:
while you are correct, that only matters in the definition, not the calling.  however, for aesthetics, i'll change this :)